media/videostitcher/get_cdn_key.go

```diff
@@ -22,7 +22,7 @@ import (
 	"io"
 
 	stitcher "cloud.google.com/go/video/stitcher/apiv1"
-	"cloud.google.com/go/video/stitcher/apiv1/stitcherpb"
+	stitcherstreampb "cloud.google.com/go/video/stitcher/apiv1/stitcherpb"
```

Comment:
FWIW adding the alias here and elsewhere for stitcherpb is not necessary (up to your discretion though).

Reply:
Ack. I added these in #3305 so I'd like to keep as-is.